hudi-client/hudi-client-common/src/main/java/org/apache/hudi/client/BaseHoodieClient.java

```diff
@@ -95,6 +98,8 @@ protected BaseHoodieClient(HoodieEngineContext context, HoodieWriteConfig client
         clientConfig.getHoodieClientHeartbeatIntervalInMs(), clientConfig.getHoodieClientHeartbeatTolerableMisses());
     this.metrics = new HoodieMetrics(config);
     this.txnManager = new TransactionManager(config, fs);
+    TrueTimeGeneratorManager.initialize(
+        new HoodieTrueTimeConfig(new TypedProperties(clientConfig.getProps())), hadoopConf);
```

Comment:
Kind of think we should have a tool method in `HoodieWriteConfig` named `HoodieWriteConfig#getTimeGeneratorConfig`, and let's rename `TrueTimeGeneratorManager` -> `TimeGeneratorManager`